x-pack/plugins/security_solution/public/common/components/event_details/summary_view.tsx

```diff
@@ -0,0 +1,167 @@
+  EuiDescriptionListDescription,
+} from '@elastic/eui';
+
+import { get, getOr } from 'lodash/fp';
+import { TimelineEventsDetailsItem } from '../../../../common/search_strategy';
+import { FormattedFieldValue } from '../../../timelines/components/timeline/body/renderers/formatted_field';
+import * as i18n from './translations';
+import { BrowserFields } from '../../../../common/search_strategy/index_fields';
+import {
+  ALERTS_HEADERS_RISK_SCORE,
+  ALERTS_HEADERS_RULE,
+  ALERTS_HEADERS_SEVERITY,
+} from '../../../detections/components/alerts_table/translations';
+import {
+  IP_FIELD_TYPE,
+  SIGNAL_RULE_NAME_FIELD_NAME,
+} from '../../../timelines/components/timeline/body/renderers/constants';
+import { DESTINATION_IP_FIELD_NAME, SOURCE_IP_FIELD_NAME } from '../../../network/components/ip';
+import { LineClamp } from '../line_clamp';
+import { useRuleAsync } from '../../../detections/containers/detection_engine/rules/use_rule_async';
+
+type Summary = Array<{ title: string; description: JSX.Element }>;
+
+const fields = [
+  { id: 'signal.status' },
+  { id: '@timestamp' },
+  {
+    id: SIGNAL_RULE_NAME_FIELD_NAME,
+    linkField: 'signal.rule.id',
+    label: ALERTS_HEADERS_RULE,
+  },
+  { id: 'signal.rule.severity', label: ALERTS_HEADERS_SEVERITY },
+  { id: 'signal.rule.risk_score', label: ALERTS_HEADERS_RISK_SCORE },
+  { id: 'host.name' },
+  { id: 'user.name' },
+  { id: SOURCE_IP_FIELD_NAME, fieldType: IP_FIELD_TYPE },
+  { id: DESTINATION_IP_FIELD_NAME, fieldType: IP_FIELD_TYPE },
+];
+
+const getDescription = ({
+  contextId,
+  eventId,
+  fieldName,
+  value,
+  fieldType = '',
+  linkValue,
+}: {
+  contextId: string;
+  eventId: string;
+  fieldName: string;
+  value?: string | null;
+  fieldType?: string;
+  linkValue?: string;
+}) => {
+  if (fieldName === 'signal.status') {
+    return (
+      <EuiBadge>
+        <FormattedFieldValue
+          contextId={`alert-details-value-formatted-field-value-${contextId}-${eventId}-${fieldName}-${value}`}
+          eventId={eventId}
+          fieldName={fieldName}
+          fieldType={fieldType}
+          value={value}
+        />
+      </EuiBadge>
+    );
+  }
+
+  return (
+    <FormattedFieldValue
+      contextId={`alert-details-value-formatted-field-value-${contextId}-${eventId}-${fieldName}-${value}`}
+      eventId={eventId}
+      fieldName={fieldName}
+      fieldType={fieldType}
+      value={value}
+      linkValue={linkValue}
+    />
+  );
+};
+
+export const SummaryViewComponent: React.FC<{
+  browserFields: BrowserFields;
+  data: TimelineEventsDetailsItem[];
+  eventId: string;
+  timelineId: string;
+}> = ({ data, eventId, timelineId, browserFields }) => {
+  const [investigationGuide, setInvestigationGuide] = useState<string | null>(null);
+
+  const ruleIdField = useMemo(() => data.find((d) => d.field === 'signal.rule.id'), [data]);
+  const ruleId = getOr(null, 'originalValue', ruleIdField);
+  const { rule: maybeRule } = useRuleAsync(ruleId);
```

Comment:
why do we need to rename it to `maybeRule`?

Reply:
as the response may be null

Reply:
hmm, I haven't seen this pattern in our code before, maybe we can just stick to the `rule`? because we have many places in our code that we are not sure that we get the value, but we haven't use `maybe` before

ok I can see that it's coming from the Exceptions modal, but I'm not sure if we should widely adopt this pattern in the rest of the app